### ISSUE TITLE
exec permission /sbin/dmsetup

### ISSUE THREAD

**Issue by faxm0dem** [alxrem/collectd-flashcache-py#1]
dmsetup needs collectd to be run as root.
I understand `dmsetup table` is only used for discovering the flashcache devices, is that correct?

**Comment by alxrem** [comment]
Yes, dmsetup is used only to map flashcache devices to disk drives.

**Comment by faxm0dem** [comment]
Right. I'll try to find an unpriviledged alternative then
